eng/pipelines/patch_release_client.txt

```diff
@@ -32,6 +32,7 @@ com.azure:azure-communication-phonenumbers # Tests owner: whisper6284, miguhern,
 com.azure:azure-communication-sms # Tests owner: besh2014, DimaKolomiiets
 com.azure:azure-communication-rooms # Tests owner: minnieliu, Mrayyan
 com.azure:azure-containers-containerregistry # Tests owner: lmolkova
+com.azure:azure-ai-content-safety # Tests owner: zhaiyutong
```

Comment:
can you reorder it alphabetically, so content-safety will be the first item in the list.

Reply:
reordered